server.js

```diff
@@ -1,5 +1,3 @@
-process.env.UV_THREADPOOL_SIZE = 100;
```

Comment:
That's exactly the issue. Probably that branch fixes it, I think as soon as it's released we can delete the explicit setting.